perllib/FixMyStreet/App/View/Web.pm

```diff
@@ -100,7 +101,7 @@ sub add_links {
     my $text = shift;
     $text = FixMyStreet::Template::conditional_escape($text);
     $text =~ s/\r//g;
-    $text =~ s{(https?://)([^\s]+)}{"<a href=\"$1$2\">$1" . _space_slash($2) . '</a>'}ge;
+    $text =~ s{([^"'])(https?://)([^\s]+)}{"$1<a href=\"$2$3\">$2" . _space_slash($3) . '</a>'}ge;
```

Comment:
This won't catch a link if it's the first thing in `$text`. Something like `(?<!["'])` is the same and is zero-width, so you then don't need the $1 in the replacement either.